docker/xud/xud.conf

```diff
@@ -0,0 +1,24 @@
+lndDir = "~/.lnd"
```

Comment:
I know I brought this up in the last iteration of this PR, but I still don't understand why this file is needed if it does nothing but specify the default, hardcoded values. XUD does not require a config file to run (at least not currently), only for overriding default values.

Reply:
Please refer to comment above.